spec/features/user/edit_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "rails_helper"
+
+feature "Edit" do
+  before do
+    create(
+      :user,
+      email: "example@gmail.com",
+      password: "123456",
+      role: "director",
+      first_name: "Bilbo",
+      last_name: "Baggins"
+    )
+  end
+
+  scenario "edit with current password" do
+    sign_in("example@gmail.com", "123456")
+    click_on "Bilbo Baggins"
+
+    fill_in "user_first_name", with: "John"
+    fill_in "user_last_name", with: "Doe"
+    fill_in "user_current_password",	with: "123456"
+
+    click_on "Update"
+
+    expect(page).to have_content("Your account has been updated successfully.")
+    expect(page).to have_content("John Doe")
+  end
+
+  scenario "edit without current password" do
+    sign_in("example@gmail.com", "123456")
+    click_on "Bilbo Baggins"
+
+    fill_in "user_first_name", with: "John"
+    fill_in "user_last_name", with: "Doe"
+
+    click_on "Update"
```

Comment:
Maybe move it to function? 🙈

Reply:
@RasimKhusaenov Maybe you fuck yourself?